### DISCUSSION TITLE
posts/25/

### DISCUSSION THREAD

**Discussion by giscus** [Flying-dragon-boxing/BlogDiscussions#1] [Announcements]
# posts/25/

首先，如果是在用Windows的话，那么最方便的方式无疑是Visual Studio+Intel oneAPI，毕竟是地表最强IDE加上x86下最适合科学计算的运行环

https://blog.rhx0820.xyz/posts/25/

**Comment by Flying-dragon-boxing** [comment]
孩子们我回来了！马上更CLion写C++感想
